### ISSUE TITLE
A question about Dexter-core

### ISSUE THREAD

**Issue by 522730312** [dexter/dexter#8]
when i run maven under /Dexter-core , there happen some error,below are output of coomand,

[ERROR] Failed to execute goal on project dexter-common: Could not resolve dependencies for project it.cnr.isti.hpc:dexter-common:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at it.cnr.isti.hpc:hpc-utils:jar:0.0.6: Failed to read artifact descriptor for it.cnr.isti.hpc:hpc-utils:jar:0.0.6: Could not transfer artifact it.cnr.isti.hpc:hpc-utils:pom:0.0.6 from/to dropbox (https://dl.dropboxusercontent.com/u/4663256/mvn-repository/): Connect to dl.dropboxusercontent.com:443 [dl.dropboxusercontent.com/74.125.39.113] failed: Operation timed out -> [Help 1]

Is there anyone can help me, is the URL    https://dl.dropboxusercontent.com/u/4663256/mvn-repository/     has problems.

**Comment by diegoceccarelli** [comment]
HI @522730312  are you in China? could it be that dropbox is blocked? 
Btw, if the problem is hpc-utils you can check it out here: 

git clone http://git.hpc.isti.cnr.it/diego/hpc-utils.git

and then run "mvn install" inside hpc-utils (note that, this is the version 0.0.7, you will have to update the version in dexter-common)

**Comment by 522730312** [comment]
Yes,I'm in China,maybe dropbox is blocked.So , could you please send the document in the  URL    https://dl.dropboxusercontent.com/u/4663256/mvn-repository .Chinese can be tokenized,I am working on it ,an i'm sure ,the tool what i use is Stanford-CoreNLP,It's Nice.

**Comment by diegoceccarelli** [comment]
@522730312  do this steps: 
1) git clone http://git.hpc.isti.cnr.it/diego/hpc-utils.git
2) cd hpc-utils 
3) open pom.xml and change <version>0.0.7 </version> in <version>0.0.6</version> 
4) run: mvn install
5) recompile dexter.

**Comment by nooralahzadeh** [comment]
HI Diego
I have updated your code in spot generation in order to consider all disambiguation links for spot ( in ExtractSpotsCLI.java) 
![extractspotscli](https://cloud.githubusercontent.com/assets/1093791/8978266/d171deb6-36a0-11e5-891a-2532ae08fee8.jpg)
